### PR TITLE
Modernize cli

### PR DESCRIPTION
**Pull request by abelsromero**
Thank you for opening a pull request and contributing to AsciidoctorJ!

Please take a bit of time giving some details about your pull request:

## Kind of change

- [ ] Bug fix
- [ ] New non-breaking feature
- [ ] New breaking feature
- [ ] Documentation update
- [ ] Build improvement
- [x] Chore

## Description

What is the goal of this pull request?
* Bump jCommander version to latest v1.81
* Modernize CLI related classes:
  * Fix deprecated calls
  * User Java8+ structs when (opinionably) applicable
  * Applied default IntelliJ Formatting

How does it achieve that?

Are there any alternative ways to implement this?

Are there any implications of this pull request? Anything a user must know?
Test pass and I have not gone far in code changes just in case.

## Issue

If this PR fixes an open issue, please add a line of the form:

Fixes #Issue 

## Release notes

Please add a corresponding entry to the file CHANGELOG.adoc

### CODE REVIEW

File: asciidoctorj-core/src/main/java/org/asciidoctor/jruby/cli/AsciidoctorInvoker.java [hunk 1]
```diff
@@ -148,34 +147,31 @@ private URLClassLoader createUrlClassLoader(List<String> classPaths) {
...
+                    System.err.println("asciidoctor: FAILED: input file(s) '"
+                            + inputFile.getAbsolutePath()
+                            + "' missing or cannot be read");
+                    throw new IllegalArgumentException(
+                            "asciidoctor: FAILED: input file(s) '"
+                                    + inputFile.getAbsolutePath()
+                                    + "' missing or cannot be read");
+                });
+
+        return inputFiles.stream()
```
Comment (abelsromero) [inline asciidoctorj-core/src/main/java/org/asciidoctor/jruby/cli/AsciidoctorInvoker.java:165]: I refactored this but the output is actually never used, maybe we should remove and just cal convert "for each" input?

Comment (robertpanzer) [reply asciidoctorj-core/src/main/java/org/asciidoctor/jruby/cli/AsciidoctorInvoker.java:165]: Yes, that would make sense. The method is also not part of any public API, so we can just remove the return value.

**Comment by robertpanzer** [comment]
This looks good to me.
If you like you can change that one thing you comment on, otherwise I would also merge it in its current state.

**Comment by abelsromero** [comment]
> If you like you can change that one thing you comment on, otherwise I would also merge it in its current state.

Yes, thanks. I can find some time today.

**Comment by abelsromero** [comment]
Done :partying_face:

**Comment by robertpanzer** [comment]
Awesome, thank you! 🙏🎉

**Pull request merged**